scenario_specifcation

```diff
@@ -0,0 +1,96 @@
+eg01-23 (Pu)
+eg01-24 (TRU)
+
+uox
+Breeders
+
+BFR 
+USE MIXER TO PRODUCE THE DESIRED RECIPE
+GOAL : CF = 1.2
+
+
+-----------------------------------------
+eg01-29 (Pu)
+eg91-30 (TRU)
+
+
+use CFR for burner recipes
+
+""
+LWR - uox
+Burners
+GOAL : CF = .8
+""
+
+
+
+
+------------------------------------------------------------------------
+FROM APPENDIX B OF FUELCYCLEEVALUATION.INL.GOV
```

Comment:
Ah, now I understand that these can rather be cited and not copy-pasted. Will delete them (both in this branch and newer branch as well)